testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/elytron/CustomCredentialSecurityFactoryImpl.java

```diff
@@ -53,7 +52,7 @@ public Credential create() throws GeneralSecurityException {
         }
     }
 
-    @Override
+    @SuppressWarnings("unused")
```

Comment:
This SuppressWarnings annotation can be removed I think. The initialize method will be called automatically for Elytron custom components.